parsers/NL.py

```diff
@@ -97,8 +97,12 @@ def fetch_production(zone_key='NL', session=None, target_datetime=None,
                             - df_consumptions_with_exchanges['NL_import'])
 
     # Fetch all production
-    productions = ENTSOE.fetch_production(zone_key=zone_key, session=r,
-                                          target_datetime=now, logger=logger)
+    if energieopwek_nl:
```

Comment:
can you add a comment (like the one you used to describe this PR) to describe why we need this?
That will help future maintainers.
Thanks!